pandas/core/array_algos/putmask.py

```diff
@@ -120,3 +120,37 @@ def _putmask_preserve(new_values: np.ndarray, new, mask: np.ndarray):
     except (IndexError, ValueError):
         new_values[mask] = new
     return new_values
+
+
+def putmask_without_repeat(values: np.ndarray, mask: np.ndarray, new: Any) -> None:
```

Comment:
might be naming `putmask_with_repeat_inline` (but can change later as well)

Reply:
having it return None should be clear enough

Reply:
sure but names matter

Reply:
yah, i just happen to like this name